openpype/plugins/publish/extract_review.py

```diff
@@ -152,7 +152,7 @@ def _get_outputs_per_representations(self, instance, profile_outputs):
             if input_ext.startswith("."):
                 input_ext = input_ext[1:]
 
-            if input_ext not in self.supported_exts:
+            if input_ext.lower() not in self.supported_exts:
```

Comment:
I would say it should be already prefilled on representation as lowered...???

Reply:
For WP it should be lowercased, but it might come in through something different (Traypublisher ?).

I don't think explicit lowercase could hurt.

Reply:
I'm a little bit concerned about the `repre["ext"]` lowering because it could be used somewhere as source of extension for input file (didn't find any)? But at the same time it's used as integrated extension so it will also be traversed to integrated file (not good)...

Reply:
Not sure I follow.
In WP I am lowercasing as customer confirmed it is accident.
In ExtractReview I see potential issue on line 937, but that's extension of review. So it could happen, that you have `image_Art_Main.PNG` (in WP it won't happen, but in Tray publisher probably might) and `review_Art_Main.png`, but I still don't see an issue there.

Reply:
After discussion removed lower here.
Using lower here would hide possibly broken representation, as we would expect both repre["ext"] and repre["name"] be lowercased. In case the aren't, review won't get created >> someone will notice and fix issues on source representation.